client/src/index.css

```diff
@@ -1097,4 +1101,8 @@ header.searchPagination {
 		display: block !important;
 		height: 100% !important;
 	}
+	/* adds random characters to printed pdf*/
```

Comment:
Should at least have a space before the comment close ` */`, but frankly I don't understand this comment…

Reply:
react-svg library was adding some elements to the end of the document which was causing empty space => empty pages.